bigtable/hbase/snippets/pom.xml

```diff
@@ -62,7 +62,7 @@
     <dependency>
       <groupId>com.google.cloud.bigtable</groupId>
       <artifactId>bigtable-hbase-1.x</artifactId>
-      <version>2.9.0</version>
+      <version>2.10.0</version>
```

Comment:
Do you need to manually set this version? Can it be pulled from the bom?

Reply:
These version shouldnt be here to begin with. I will remove them in a follow up PR, but I didnt want to expand scope here